architecture/core_concepts/routes.adoc

```diff
@@ -742,9 +743,9 @@ spec:
 <1> The name of the object, which is limited to 63 characters.
 <2> The `*termination*` field is set to `reencrypt`. Other fields are as in edge
 termination.
-<3> The `*destinationCACertificate*` field specifies a CA certificate to
-validate the endpoint certificate, securing the connection from the router to
-the destination. This field is required, but only for re-encryption.
+<3> The `*destinationCACertificate*` field is required for re-encryption, and
```

Comment:
The cert is only required if the admin didn't configure a default serving certificate.  Instead would say:

> ... is required for re-encryption, ..., securing the connection from the router to the destination pods.  If the service is using a service signing certificate, or the administrator has specified a default CA certificate for the router and the service has a certificate signed by that CA, this field may be omitted.